src/composables/multisigAccounts.ts

```diff
@@ -107,30 +121,29 @@ export function useMultisigAccounts({ store }: IDefaultComposableOptions) {
             gaAccountId ? sdk.balance(gaAccountId) : 0,
           ]);
 
-          const consensus = consensusResult.decodedResult as any;
+          const consensus: IMultisigConsensus = camelCaseKeysDeep(consensusResult.decodedResult);
```

Comment:
This will lead to an error, `txHash` is breaking in this case.

Reply:
![image](https://user-images.githubusercontent.com/9008074/220082846-567c9b2d-6bbf-4876-a1dd-56557fda8f6f.png)
First initial value, second is after `camelcaseKeysDeep`

Reply:
Thanks for the clarification. Please take a look on my change. I console.log'ged it and looks as it should be. But maybe you see more issues?